neqo-server/src/main.rs

```diff
@@ -90,6 +90,10 @@ struct Args {
     /// Use http 0.9 instead of HTTP/3
     use_old_http: bool,
 
+    #[structopt(name = "max-streams", long)]
+    /// Set MAX_STREAMS limits.
+    max_streams: Option<u64>,
```

Comment:
This isn't enough, as it doesn't allow you to control all of the different limits independently.  We should at least have an idea of how we want to manage that.